config/lobby/db/001_create_tables

```diff
@@ -1,94 +1,58 @@
-create table bad_words (
-    word character varying(40) not null
+CREATE TABLE ta_users (
+    username character varying(40) NOT NULL PRIMARY KEY,
+    password character(34),
+    bcrypt_password character(60),
+    CHECK (password IS NOT NULL OR bcrypt_password IS NOT NULL),
+    email character varying(254) NOT NULL,
+    joined timestamp with time zone DEFAULT CURRENT_TIMESTAMP,
+    lastlogin timestamp with time zone DEFAULT CURRENT_TIMESTAMP,
+    admin integer NOT NULL DEFAULT 0
```

Comment:
Left this the same for now, but I'd suggest using either a boolean type (if we only need to roles), or enums (if we want to add new roles in the future) in the future.

Reply:
Pls avoid enumerated types in DB. They are awful in mysql, I don't know if postgres has the same issues but I can't imagine it being much better. A second table and foreign key constraint is the classic way to do that. In this case a boolean value column seems the most straight forward.

Reply:
Agree a boolean type would be better here

Reply:
Ok, converting this to a boolean in this PR, I noticed the value us being retrieved as boolean anyways.
Would you mind telling me what is wrong with enum types in mysql?
It has probably been a year since I last used them.
Looking at the mysql documentation it seems they are pretty similar with the exception that in psql enum types need to be defined in a seperate statement before using them.

Reply:
> Would you mind telling me what is wrong with enum types in mysql?

I'm not the best to describe why. My former boss who did mysql for decades hated them with a passion. I think because the encoding is not just a simple string, so you have massive backward compatibility issues introduced. Most of the time the data structure should just be captured with a constraint check or a second table. So part of the problem is that there are other better solutions for an 'enum' type without the problems.

Reply:
IMO, put that update in its own branch.

Reply:
Was a 2 line change... I think I'm going to revert the password changes and submit a follow-up PR for them and keep everything else in here

Reply:
Thanks, there is quite a bit going on. We need to be sure there is very thorough testing, splitting things up will help ensure everything gets its due course.